lib/utility.js

```diff
@@ -0,0 +1,135 @@
+const moment = require('moment');
+// const queries = require('../db/queries');
+
+const isValidDateTime = (timestamp) => {
+  return moment(timestamp, moment.ISO_8601, true).isValid();
+};
+
+const isInt = (val) => {
+  return val === +val && val === (val | 0);
+};
+
+// const isFloat = (val) => {
+//   return val === +val && val !== (val | 0);
+// };
+
+const hasValidTickData = ({ instrument, time, bid, ask, bid_vol, ask_vol }) => {
+  return !!instrument && !!time && !!bid && !!ask && !!bid_vol && !!ask_vol &&
+    (instrument.length === 6) &&
+    !Number.isNaN(parseFloat(bid)) &&
+    !Number.isNaN(parseFloat(ask)) &&
+    isInt(+bid_vol) &&
+    isInt(+ask_vol) &&
+    isValidDateTime(time);
+};
+
+const parseDateToUnixTime = (dateString) => {
+  return Date.parse(dateString);
+};
+
+const parseDateToUnixTime5S = (dateString) => {
+  const unix = parseDateToUnixTime(dateString);
+  return unix - (unix % 5000);
+};
+
+const computeOHLCFromTicks = (ticksData, unix) => {
+  const dt = moment(Number.parseInt(unix, 10)).toISOString();
+  const ticks = ticksData.length;
+  const { id_pairs } = ticksData[0];
```

Comment:
Consider declaring these variables in one statement:
let minDt, maxDt, bid_o, bid_c...